docs/install.rst

```diff
@@ -113,16 +113,20 @@ Installing Rally
 ----------------
 
 1. Ensure ``~/.local/bin`` is in your ``$PATH``.
-2. Install Rally: ``python3 -m pip install --user esrally``.
+2. Ensure pip is the latest version: ``python3 -m pip install --user --upgrade pip``
+3. Install Rally: ``python3 -m pip install --user esrally``.
 
 VirtualEnv Install
 ------------------
 
 You can also use Virtualenv to install Rally into an isolated Python environment without sudo.
 
-1. Set up a new virtualenv environment in a directory with ``python3 -m venv .``
-2. Activate the environment with ``source /path/to/virtualenv/dir/bin/activate``
-3. Install Rally with ``python3 -m pip install esrally``
+1. Set up a new virtualenv environment in a directory with ``python3 -m venv .venv``
+2. Activate the environment with ``source /path/to/virtualenv/.venv/bin/activate``
```

Comment:
I wonder whether we could simplify this path here assuming we are already in the current directory in the step above, i.e.

```
source .venv/bin/activate
```

Reply:
We could, I think we have it here because a few lines after we reference this step at a point where the user might not be in the same directory anymore:

> Whenever you want to use Rally, run the activation script (step 2 above) first.

Reply:
Good point. Then let's be safe here.